.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -88,40 +88,47 @@ steps:
   #     Task variable SIG_FOR_PRODUCTION is set to True and passed to the following steps.
   #     Built sig will be deleted because it has been converted to VHD, and thus not needed.
   - bash: |
+      set -e
```

Comment:
I also changed this script to be a strange single liner script to a normal script with a "set -e" to get it to exit if anything goes wrong.